### ISSUE TITLE
Could not find a declaration file for module 'fictional'.

### ISSUE THREAD

**Issue by noook** [oftherivier/fictional#37]
It seems that the types are not correctly exported for the module

`Could not find a declaration file for module 'fictional'.`

info: 
Package manager: pnpm@8.10.2
Node version: v18.18
```json
// package.json
{
  "name": "app",
  "type": "module",
  "dependencies": {
    "fictional": "^2.0.0",
  },
}
```

**Comment by justinvdm** [comment]
Hi @noook, I've set up a test project with this same package.json, node version and pnpm version, and I still seem to have the types working.

I'd like to figure this out though. My next guess would be some other difference between our environments. Are you seeing this error in vscode? If so, could I see which entries you have in your vscode `settings.json` for keys starting with `typescript`?

**Comment by noook** [comment]
I made it happen within a Nuxt 3 project. I read something very luckily on Twitter saying that the `exports` field of package.json was not correctly filled on projects and I took my shot at it, turns out it was just missing the "types" in the exports field.

**Comment by noook** [comment]
@justinvdm  I made it happen within a Nuxt 3 project. I read something very luckily on Twitter saying that the `exports` field of package.json was not correctly filled on projects and I took my shot at it, turns out it was just missing the "types" in the exports field.

**Comment by justinvdm** [comment]
Thank you @noook! Released as v2.0.1.
